utils/filters.py

```diff
@@ -27,3 +27,12 @@ def filter_ignored_users(self, msg, cmd, args, dry_run):
         if msg.frm.nick in self.bot_config.IGNORE_USERNAMES:
             return None, None, None
         return msg, cmd, args
+
+    @cmdfilter
+    def filter_private_diverts(self, msg, cmd, args, dry_run):
+        """
+        Leave a message that response is diverted as private message.
+        """
+        if cmd in self.bot_config.DIVERT_TO_PRIVATE:
+            self.send(msg.frm, 'It is diverted into private message, try sending this command directly to @co-robo')
+        return msg, cmd, args
```

Comment:
this code is working fine but it is blocked due to an issue in errbot

[errbotio/errbot#1071](https://github.com/errbotio/errbot/issues/1071)